hr_governance/models/governance_circle.py

```diff
@@ -0,0 +1,638 @@
+# Copyright 2025 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import random
+
+from odoo import Command, _, api, fields, models
+from odoo.exceptions import AccessError, UserError
+from odoo.tools import html_to_inner_content, ormcache
+from odoo.tools.misc import str2bool
+
+
+class GovernanceCircle(models.Model):
+    _name = "governance.circle"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _description = "Governance Circle and Roles"
+    _parent_store = True
+
+    name = fields.Char(
+        tracking=True,
+        required=True,
+        readonly=False,
+        compute="_compute_fields_from_type",
+        store=True,
+    )
+    parent_id = fields.Many2one(
+        "governance.circle",
+        ondelete="restrict",
+        index=True,
+        domain="[('is_circle', '=', True)]",
+    )
+    parent_path = fields.Char(index=True)
+    child_ids = fields.One2many(
+        "governance.circle",
+        "parent_id",
+    )
+    # assigned members for circle
+    member_ids = fields.Many2manyCustom(
+        "hr.employee",
+        "governance_circle_member_rel",
+        "circle_id",
+        "member_id",
+        create_table=False,
+        string="Members",
+        compute="_compute_member_ids",
+    )
+    # assigned members for role
+    member_rel_ids = fields.One2many("governance.circle.member.rel", "circle_id")
+    assigned_user_ids = fields.Many2many(
+        "res.users", compute="_compute_assigned_user_ids", string="Assigned Users"
+    )
+    member_cnt = fields.Integer(compute="_compute_member_cnt")
```

Comment:
```suggestion
    member_count = fields.Integer(compute="_compute_member_cnt")
```

there's point in skipping these 2 characters here. It only hurts readability and it's against odoo codebase conventions